src/FilterProviderInterface.php

```diff
@@ -1,23 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Laminas\Filter;
-
-use Laminas\ServiceManager\ServiceManager;
-
-/**
- * Implement this interface within Module classes to indicate that your module
- * provides filter configuration for the FilterPluginManager.
```

Comment:
But doesn't this also apply to a config provider? What I mean by that is, wouldn't the interface also help the user in a config provider?